src/mono/netcore/System.Private.CoreLib/src/System/Reflection/CustomAttribute.cs

```diff
@@ -591,6 +591,8 @@ internal static bool IsDefined(ICustomAttributeProvider obj, Type attributeType,
         [MethodImplAttribute(MethodImplOptions.InternalCall)]
         private static extern bool IsDefinedInternal(ICustomAttributeProvider obj, Type AttributeType);
 
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2006:UnrecognizedReflectionPattern",
+            Justification = "Custom attributes are analyzed by the linker")]
```

Comment:
The justification is weird - maybe a bit more detail (linker analyzes all properties and marks them?)